.github/workflows/ci.yaml

```
@@ -0,0 +1,14 @@
+name: ci
+
+on:
+  pull_request:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@0a12ed9d6a96ab950c8f026ed9f722fe0da7ef32 # v5.0.2
```

```suggestion
      - uses: actions/setup-go@v5.0.2
```
dependabot doesn't create update PRs for actions that use the hash instead of a version number.

---

Dr. Empirical disagrees: https://github.com/ko-build/setup-ko/commit/5db88cba55160b19e13be1d2b74f23390271bd14